be/src/exprs/runtime_filter_rpc.cpp

```diff
@@ -39,7 +39,12 @@ struct IRuntimeFilter::rpc_context {
 Status IRuntimeFilter::push_to_remote(RuntimeState* state, const TNetworkAddress* addr) {
     DCHECK(is_producer());
     DCHECK(_rpc_context == nullptr);
-    PBackendService_Stub* stub = state->exec_env()->brpc_stub_cache()->get_stub(*addr);
+    std::shared_ptr<PBackendService_Stub> stub(
+            state->exec_env()->brpc_stub_cache()->get_stub(*addr));
+    if (!stub) {
+        LOG(WARNING) << "Get rpc stub failed, host=" << addr->hostname << ", port=" << addr->port;
+        return Status::InternalError("get rpc stub failed");
```

Comment:
Should return more useful info in error log, like ip and reason

Reply:
why get failed is logged in state->exec_env()->brpc_stub_cache()->get_stub

Reply:
done